portfolio/src/main/webapp/index.html

```diff
@@ -7,13 +7,98 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <!-- <img src="sunset.webp" alt="sunset Picture" -->
+      <!-- <p style="background-image: url('sunset.webp');"> -->
+    <div id="content" ">
+      <div id="whiteCenter">
+        <img src="Images/layout.webp" alt="My picture" width="1100" height="2000" />
+        </div>
+      <h1><i>My Portfolio</i></h1>
+      <a href="https://www.linkedin.com/in/muhammad-umair-07ab911b3/"
+        target="_blank" class="link1">
+        <button class="linkedIn">LinkedIn</button>
+      </a>
+      <a href="https://github.com/mumair989"
+      target="_blank">
+      <button class="gitHub">GitHub</button>
+    </a>
+    <a href="#contact"><button class="Contact">Contact</button></a>
+    <a href="#projects"><button class="ProjectButton">Projects</button></a>
+    <a href="#secondBox"><button class="WorkButton">Experience</button></a>
+
+      
+      <!-- <p id="heading" >Hi there!</p> -->
+      <div id="image1">
+      <img src="Images/image1.PNG" alt="My picture" width="250" height="350" />
+      </div>
+<p id="sideNote"><i style="color:rgb(255, 0, 0)">Hi their!</i><br> I am Muhammad Umair <br> </p>
+
+<article id="about">
+  <h2 id="aboutHeader">About Me</h2>
+      <p id="myLife">My name is <i style="color:red;">Muhammad Umair</i>, I live in New York City, I am really excited to be a participant of Google SPS Program.
+        I joined Google SPS because I wanted to meet people from across the world and learn new things from everyone and the program itself.
+        I believe SPS will further develop my technical and interpersonal skills and hopefully I will create something really cool together.</p>
+      </article>
+     
+        <div id="projects">
+        <h2 id="projectHeader">Projects</h2>
```

Comment:
[3.1.9](https://google.github.io/styleguide/htmlcssguide.html#id_Attributes): Avoid unnecessary id attributes.

I'm noticing that `#projectHeader`, `#aboutHeader` and `#Work` are essentially styled the same way and function the same way. This is where a common CSS class comes in handy because then you can change the style for all of them at the same time instead of having to copy/paste the styles multiple times.

IDs are generally used for anchoring elements, so if you add `#projects` to the address in the URL bar for example, it'll scroll the page down to this section. However, it's better to use classes for styling elements instead as they can be reused across multiple elements.